src/NHibernate.Test/Linq/ProjectionsTests.cs

```diff
@@ -241,6 +266,19 @@ public void ProjectAnonymousTypeWithCollection()
 			Assert.That(result[0].o.OrderLines, Is.EquivalentTo(result[0].OrderLines));
 		}
 
+		[Test]
+		public void ProjectAnonymousTypeWithCollectionWithFetch()
+		{
+			// NH-3333
+			// done by WCF DS: context.Orders.Expand(o => o.OrderLines) from the client 
```

Comment:
Above comments seems irrelevant for added test cases (applies to other added tests too)